cloudformation/panther-cloudwatch-firehose.yml

```diff
@@ -108,6 +108,7 @@ Resources:
         BufferingHints:
           IntervalInSeconds: !Ref BufferIntervalInSeconds
           SizeInMBs: 128 # Maximum
+        CompressionFormat: GZIP
```

Comment:
The Gzip needs to be removed. Cloudwatch subscriptions are compressed by default, setting compression on firehose creates a double compression. Explanation [here](https://aws.amazon.com/kinesis/data-firehose/faqs/)

Reply:
I think this was an artifact of the commit that I initially branched out from. I can't imagine why I would've been editing this part.

Reply:
I know what happened: was trying to sync internal changes to this and confused `panther-cloudwatch-firehose.yml` and `panther-fluentd-firehose.yml`

Does this mean it should be removed [here](https://github.com/panther-labs/panther-auxiliary/blob/main/cloudformation/panther-fluentd-firehose.yml#L28), too?

Reply:
Does not need to be removed from that link. Only an issue when using cloudwatch subscriptions that we do not need a GZIP compression on firehose side. Any other use of Firehose like writing from Fluentd can still leverage the firehose compression.